buildlib/azure-pipelines-release.yml

```diff
@@ -22,13 +22,31 @@ stages:
           - bash: ./autogen.sh
             displayName: Setup autotools
 
+          - bash: |
+              sed -i -e 's/@VERSION@-SNAPSHOT/@VERSION@/g' bindings/java/pom.xml.in
+              sed -i -e 's/@VERSION@-SNAPSHOT/@VERSION@/g' bindings/java/src/main/native/Makefile.am
+            displayName: Set JUCX release version
+
           - bash: |
               set -eE
               gcc --version
-              ./contrib/configure-release
+              ./contrib/configure-release --with-java
```

Comment:
This will publish pre-configured tarball on GitHub with Java support. @yosefe is that the intention? If not, let's split publish jobs into two: general (same as before) and JUCX one. They will go in parallel, no time overhead.